en/orm/retrieving-data-and-resultsets.rst

```diff
@@ -77,6 +77,13 @@ by using the ``finder`` option::
     $article = $articles->get($id, [
         'finder' => 'translations',
     ]);
+    
+    The list of options supported by get() are:
```

Comment:
It actually supports all the options supported in `$options` argument of `find()` methods, not just these. While some of them like `order` wouldn't have any practical value others like `join` would.

Reply:
so additional conditions like   `soft_deleted_column` = 0 can be provided. 

Shall I make another PR ?

Reply:
`get()` supports `key`, `cache`, `finder` options + all options supported by `find` or specified finder.